eth2/types/src/spec/few_validators.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{ChainSpec, Slot};
```

Comment:
For consistency, should we put this inside `mod.rs`, or split everything out into `chain_spec.rs`. Either way combine the struct with the implementations. 
I guess this goes with `foundation.rs` as well. 
I see the benefit of separating as they show distinct logic, but like we did with `beacon_chain.rs` I think combining the impls with the structs throughout the project is good. 
Alternatively, having a directory structure like:
```
chain_spec.rs
chain_spec/
   foundation.rs
   few_validators.rs
```
could be useful for structs with large impl's.

Reply:
Yeah good call. I think keeping them in the same file here works -- `few_validators()` uses logic from `foundation()` so it's reasonable they live together.

This is a relic from before we standardized.